amethyst_renderer/src/pass/sprite/interleaved.rs

```diff
@@ -281,8 +281,12 @@ impl SpriteBatch {
 
             let dir_x = transform.x * sprite_data.width;
             let dir_y = transform.y * sprite_data.height;
-            let pos =
-                transform * Vector4::new(sprite_data.offsets[0], sprite_data.offsets[1], 0.0, 1.0);
+            let pos = transform * Vector4::new(
+                -sprite_data.offsets[0],
```

Comment:
Coordinates are still starting at the bottom left and going towards the top right?

Reply:
negating doesn't really make sense to me

Reply:
Yeap they're still bottom left to top right.

The negation means shift the sprite left and down, I don't mind not negating it and leaving that to the user; I don't know if there's a common convention.

On my phone now so I'll change it when I get to my computer

Reply:
Pushed! Also updated docs, pong examples + tutorial, and sprites_ordered example

Reply:
oohh.. that's supposed to be the pivot point heh

Reply:
Yeaps! does it change your preference? (it's easy to remove, can just pop the last commit)

Reply:
I would prefer either rename to `pivot` and keep the old one, or go with the current change.
But probably on others to decide,

Reply:
I'll keep it current unless anyone else *really* wants it negated, it makes sense in my head now :stuck_out_tongue:

Reply:
I think I didn't understand what this is about. What was the negation actually doing to the pivot? I'm expecting it to default in the center of the sprite and when you offset it the positive coordinates go towards the top right.

Reply:
Hoping I understand what you're saying, so I'll try clarify before and after:

* Previously (before sprite batching), `offsets` meant "how many pixels to shift the sprite left and down", so:

    - When offsets were `0, 0`, the sprite is drawn like:

        ```text
          |ABCD| // ABCD indicates the top of the sprite
          |    |
          |    |
          x----| // x marks the entity position
        ```

    - When offsets were `1, 2`, the sprite is drawn like:

        ```text
         |ABCD|
         |x   | <- the sprite is shifted 2 pixels down relative to the entity
         |    |
         |----|

          ^ and one pixel to the left
        ```

* After sprite batching, instead of specifying `1, 2`, you have to specify `-1, -2` to get the same effect. I did revert this change when I opened the PR, but then was convinced that it makes sense to not negate the offsets, so undid the revert (i.e. redid it to mean you should specify `-1, -2` when you want to shift it left and down.)

There wasn't a default for "use `sprite_w / 2.0` and `sprite_h / 2.0`" when creating a `Sprite` structure directly, but the `with_sprite` function used to do this when sprites were still drawn using a `Mesh` component (before the `DrawSprite` pass was implemented).

Reply:
Please add a comment explaining it here. Thanks